### DISCUSSION TITLE
Simplified Chinese

### DISCUSSION THREAD

**Discussion by agray424** [rsms/inter#439] [Q&A] [answered]
Hi Ramus, just wanted to clarify whether your Inter typeface supports simplified Chinese character set. If you could confirm I would be very much obliged. 

Many thanks, Andy

**Comment by ThomasRettig** [comment, marked as answer]
It doesn't support Chinese characters.

**Reply by agray424** [reply]
Okay Ramus, many thanks for letting me know, much appreciated

**Reply by agray424** [reply]
Oops! apologies @ThomasRettig

**Comment by lxgw** [comment]
This font family is a western-language font family and doesn't contain any Chinese characters (Hanzi / Hanja / Kanji). However, You can match this font to some modern CJK sans-serif fonts such as [Source Han Sans](https://github.com/adobe-fonts/source-han-sans).
